packages/frontend/src/app/components/TrustRegistryTable.tsx

```diff
@@ -0,0 +1,72 @@
+'use client'
```

Comment:
Why is it a client?

Reply:
Because of `onClick={() => router.push(`/${item.id}`)}` but I'll use link as you wrote below.